### PR TITLE
Spw 992 csv does not work (SPW-1007)

### PR DESCRIPTION
**Pull request by dejakob**
http://team.showpad.com:8008/browse/SPW-1007
http://master.showpad.dev:8080/job/AngularTags-SPW-992-csv-does-not-work/

Feel free to ignore build or dist files while reviewing

### CODE REVIEW

File: src/tags-input.js [hunk 1]
```diff
@@ -340,14 +343,17 @@ tagsInput.directive('tagsInput', function($timeout, $document, $window, tagsInpu
...
-                                var match = tag.match(regex);
-
-                                if (match) {
-                                    tagList.addText(match[2]);
-                                    foundMatch = true;
-                                }
+                                regularExpressions.forEach(function (regex) {
+                                    var match = tag.match(regex);
+
+                                    if (match) {
```
Comment (klaascuvelier) [inline src/tags-input.js]: better check for this please
Adding in some comments wouldn't hurt

File: test/tags-input.spec.js [hunk 2]
```diff
@@ -648,6 +648,16 @@ describe('tags-input directive', function() {
             expect($scope.tags).toEqual([{ text: 'krishan.gupta@showpad.com' }]);
         });
 
+        it('should paste when copied data is in CSV format', function () {
```
Comment (klaascuvelier) [inline test/tags-input.spec.js]: why don't you add another test for the other regexp?

Comment (dejakob) [inline test/tags-input.spec.js]: I could defenitely do that

**Comment by dejakob** [comment]
There is still much place for improvements in this project, but I fixed the folder structure and angular module calls for this ticket...

File: Gruntfile.js [hunk 3]
```diff
@@ -22,15 +22,16 @@ module.exports = function(grunt) {
...
-                    'src/constants.js',
-                    'src/util.js',
-                    'src/init.js',
-                    'src/tags-input.js',
-                    'src/auto-complete.js',
-                    'src/transclude-append.js',
-                    'src/autosize.js',
-                    'src/bind-attrs.js',
-                    'src/configuration.js'
+                    'src/config/src/constants.config.js',
```
Comment (klaascuvelier) [inline Gruntfile.js]: why is this file here?
Bad merge?

This should be gone!

**Comment by klaascuvelier** [comment]
Add in your jenkins build: http://master.showpad.dev:8080/job/AngularTags-SPW-992-csv-does-not-work/

File: Gruntfile.js (hunk 3 above)
Comment (dejakob) [inline Gruntfile.js]: Funny thing, it should be removed
